app/views/govuk_publishing_components/components/docs/big_number.yml

```diff
@@ -0,0 +1,75 @@
+name: Big number
+description: A big number, with a label if needed, for visualising quantitative values such as the number of government departments or historical prime ministers.
```

Comment:
It looks like that the component is only meant to display positive integers. So either you should specify that constraint and create tests for when something else is passed, or you should allow more types of numbers (looking at the code it might actually work if you pass any string, in fact)